src/System.Text.Json/src/System/Text/Json/JsonEncodedText.cs

```diff
@@ -34,7 +34,7 @@ private JsonEncodedText(byte[] utf8Value)
         /// <summary>
         /// Encodes the string text value as a JSON string.
         /// </summary>
-        /// <param name="value">The UTF-16 encoded value to be transformed as JSON encoded text.</param>
+        /// <param name="value">The value to be transformed as JSON encoded text.</param>
```

Comment:
Can you update the following instances as well (within `Utf8JsonReader`/`JsonSerializer`):
https://github.com/dotnet/corefx/blob/0b8a72e7c8b57a753dbeab762b3967ecda6d47c7/src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs#L23-L26

https://github.com/dotnet/corefx/blob/0b8a72e7c8b57a753dbeab762b3967ecda6d47c7/src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs#L49-L52

https://github.com/dotnet/corefx/blob/0b8a72e7c8b57a753dbeab762b3967ecda6d47c7/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs#L306-L309

cc @bartonjs, @steveharter

Reply:
Done!